app/views/users/_form.html.erb

```diff
@@ -11,28 +11,42 @@
     </div>
   <% end %>
   <br>
-
-  <div class="field">
-    <%= f.label :first_name %>
-    <%= f.text_field :first_name %>
+<form class="form-horizontal">
```

Comment:
@stevefake what's the purpose of the form within the form, is it going to be hitting a separate endpoint?